django/views/defaults.py

```diff
@@ -52,6 +52,8 @@ def page_not_found(request, exception, template_name=ERROR_404_TEMPLATE_NAME):
         if template_name != ERROR_404_TEMPLATE_NAME:
             # Reraise if it's a missing custom template.
             raise
+        # Rendering template even though there are no substitutions allows
```

Comment:
`# Render template (even though there are no substitutions) to allow inspecting the context in tests.`